Formula/samba.rb

```diff
@@ -140,7 +134,7 @@ def caveats
     EOS
 
     port = free_port
-    spawn smbd, "-S", "-F", "--configfile=smb.conf", "--port=#{port}", "--debuglevel=4", in: "/dev/null"
+    spawn smbd, "-F", "--configfile=smb.conf", "--port=#{port}", "--debuglevel=4", in: "/dev/null"
```

Comment:
```suggestion
    spawn smbd, "--debug-stdout", "-F", "--configfile=smb.conf", "--port=#{port}", "--debuglevel=4", in: "/dev/null"
```
Maybe replace this with --debug-stdout instead (https://github.com/samba-team/samba/commit/3467214c) so that failed tests have some output for debugging?

Reply:
Fixed